examples/models/deepseek/v4/deepseek_v4_decode_swa_draft.py

```
@@ -0,0 +1,303 @@
+        "x": tensors["x_hc"],
+        "hc_fn": tensors["hc_attn_fn"],
+        "hc_scale": tensors["hc_attn_scale"],
+        "hc_base": tensors["hc_attn_base"],
+        "x_mixed": x_mixed,
+        "post": post_t,
+        "comb": comb_t,
+    })
+
+    # ===== Attention.forward (model.py:484-543), ratio==0 branch =====
+    start_pos = int(tensors["start_pos"])
+    bsz, seqlen, _ = x_mixed.shape
```

Ruff (0.15.12)

[warning] Unpacked variable `bsz` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)